docker/insgorund-entrypoint.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+export IP=`awk 'END{print $1}' /etc/hosts`
```

Comment:
Are you sure that this trick works exactly as expected? E.g. on my Raspberry Pi it returns 127.0.1.1. If I remember correctly insolard and insgorund are not necessarily deployed on the same node. Thus maybe an IP should be actually an argument of this script.

Reply:
It's everything about docker container. It's one way to obtain docker public ip of local container for raw docker